ethcore/client-traits/src/lib.rs

```diff
@@ -423,3 +424,57 @@ pub trait ProvingBlockChainClient: BlockChainClient {
 	/// Get an epoch change signal by block hash.
 	fn epoch_signal(&self, hash: H256) -> Option<Vec<u8>>;
 }
+
+/// Format for importing/exporting blocks
+#[derive(Debug, PartialEq)]
+pub enum DataFormat {
```

Comment:
This is not the right place for type definitions. Can you move this to `common-types` please?